### DISCUSSION TITLE
Copilot shows much more often its context

### DISCUSSION THREAD

**Discussion by K0IN** [community/community#47967] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Since the last couple of days copilot starts showing its context much more often

I often (like a lot) get autocompletes like this:

// Path: src\some-project\interfaces\command.ts
// Compare this snippet from src\some-project\services\filesearch.ts:
// bla bla bla (my code)

This happens in both python (much more often) and typescript, also in inline completions as well as the sidepanel.

**Comment by iamcjwilliams** [comment]
@K0IN Thank you for your feedback, our copilot team has been notified

**Reply by emmadesilva** [reply]
This happens to me a lot, and has for pretty much the entire year I've used Copilot. Is there any fix on the horizon?

**Reply by RaYVen1337** [reply]
Happens to me a lot as well.
edit: using jetbrains webstorm plugin.
